torch/nn/modules/module.py

```diff
@@ -247,23 +247,23 @@ def forward(self, x):
     training: bool
     _is_full_backward_hook: Optional[bool]
 
-    def __init__(self):
+    def __init__(self) -> None:
         """
         Initializes internal Module state, shared by both nn.Module and ScriptModule.
         """
         torch._C._log_api_usage_once("python.nn_module")
 
         self.training = True
-        self._parameters = OrderedDict()
-        self._buffers = OrderedDict()
-        self._non_persistent_buffers_set = set()
-        self._backward_hooks = OrderedDict()
+        self._parameters: Dict[str, Optional[Parameter]] = OrderedDict()
+        self._buffers: Dict[str, Optional[Tensor]] = OrderedDict()
+        self._non_persistent_buffers_set: Set[str] = set()
+        self._backward_hooks: Dict[int, Callable] = OrderedDict()
         self._is_full_backward_hook = None
```

Comment:
This one should be an Optional[bool] I think

Reply:
It is (and you've annotated it ;)), see 
https://github.com/pytorch/pytorch/blob/287603f51cd0303f267f1f978bcab71aa07e3d77/torch/nn/modules/module.py#L248